ambry-router/src/test/java/com.github.ambry.router/MockServer.java

```diff
@@ -63,6 +64,7 @@
   private boolean getErrorOnDataBlobOnly = false;
   private final ClusterMap clusterMap;
   private final String dataCenter;
+  private final HashMap<RequestOrResponseType, Integer> requestCounts = new HashMap<>();
```

Comment:
Yeah, I didn't think this needs to be concurrent - but just made it one to be safe.